dew/storage.py

```diff
@@ -13,8 +13,22 @@ class StorageController(object):
     def __init__(self, path):
         self.path = path
 
+    def _makedirs(self, p):
+        os.makedirs(p, exist_ok=True)
+        if os.name == 'nt':
+            import ctypes
+            def _ensurehidden(p):

```

Comment:
code style: This function doesn't need to start with an underscore because it's inline. I'm using underscores for class methods that aren't intended to be used publicly, but this rule doesn't apply to inline methods, because there's no way to actually access them outside of the method.